src/toil/batchSystems/kubernetes.py

```diff
@@ -943,10 +981,13 @@ def executor():
 
     logging.basicConfig(level=logging.DEBUG)
     logger.debug("Starting executor")
+    
+    # If we don't manage to run the child, what should our exit code be?
```

Comment:
I picked this because it's already (now) specified as what get_updated_batch_job ought to return if it doesn't have an exit code for the job. I figured the case of being able to start the pod but not whatever the user asked for ought to be similar to other weird conditions like the pod vanishing.